ejb3/src/main/java/org/jboss/as/ejb3/subsystem/EJB3SubsystemRootResourceDefinition.java

```diff
@@ -95,6 +95,14 @@
                     .setFlags(AttributeAccess.Flag.RESTART_NONE)
                     .build();
 
+    static final SimpleAttributeDefinition DEFAULT_STATEFUL_BEAN_SESSION_TIMEOUT =
+            new SimpleAttributeDefinitionBuilder(EJB3SubsystemModel.DEFAULT_STATEFUL_BEAN_SESSION_TIMEOUT, ModelType.LONG, true)
+                    .setXmlName(EJB3SubsystemXMLAttribute.DEFAULT_SESSION_TIMEOUT.getLocalName())
+                    .setAllowExpression(true) // we allow expression for setting a timeout value
+                    .setValidator(new LongRangeValidator(-1, Integer.MAX_VALUE, true, true))
+                    .setFlags(AttributeAccess.Flag.RESTART_NONE)
```

Comment:
RESTART_NONE implies that "A modification to the attribute can be applied to the runtime without requiring a restart".  However, in this implementation, modifications to this attribute do not apply to deployed EJBs without redeploying the application.  This should probably be noted in the attribute description, otherwise we set a false expectation to users.

Reply:
Added to attribute description.